nurseconnect/services.py

```diff
@@ -10,7 +10,9 @@ def get_clinic_code(clinic_code):
     if settings.FAKE_CLINIC_CODE_VALIDATION and settings.DEBUG:
         return [0, 1, "fake_clinic_name"]
 
-    url = settings.CLINIC_CODE_API
+    url = "{}?criteria=value:{}".format(
+        settings.CLINIC_CODE_API,
+        clinic_code)
```

Comment:
Can we not rather have `requests` handle the querystring generation? http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls .

That would avoid things breaking for weird characters that a user can put in